features/adults/conviction_military.feature

```diff
@@ -9,7 +9,7 @@ Feature: Conviction
     Then I should see "<known_date_header>"
 
     And I enter a valid date
-    Then I should see "<length_type_header>"
+    Then I should see "Was the length of the order given in weeks, months or years?"
```

Comment:
I think this also needs to be customised, `Was the length of the detention [...]` 🤣